From 4623e36006c1428e876c9a326b43f713cfec658e Mon Sep 17 00:00:00 2001
From: David Herrmann <dh.herrmann@gmail.com>
Date: Sat, 11 Oct 2014 18:53:26 +0200
Subject: [PATCH] NEWS: don't mention noinst_* debug programs

systemd-evcat and systemd-modeset are not installed systemd wide and
should really just be used by developers to test the libsystemd-terminal
features. No reason to mention it in NEWS.. despite the undulating
rainbows..
---
 NEWS | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/NEWS b/NEWS
index 6d93787bf8..c18ceaac5e 100644
--- a/NEWS
+++ b/NEWS
@@ -37,13 +37,6 @@ CHANGES WITH 217:
           generator will parse the resume= option on the kernel
           command-line to trigger resume.
 
-        * systemd-evcat input debugging tool has been added. It will
-          print keys read for the input system and show modifier
-          status.
-
-        * systemd-modeset graphics debugging tool will show undulating
-          rainbows on all connected displays.
-
         * A user console daemon systemd-consoled has been added. It is
           a preview, and will so far open a single terminal on each
           session of the user marked as Desktop=SYSTEMD-CONSOLE.
